src/test/deploy/functions/prepare.spec.ts

```diff
@@ -159,4 +167,71 @@ describe("prepare", () => {
       expect(beforeSignIn.blockingTrigger.options?.refreshToken).to.be.false;
     });
   });
+
+  describe.skip("prepare", () => {
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    it.skip("should be tested", () => {});
+  });
```

Comment:
Fun fact: `describe("prepare")` should create a skipped block for you without having to include all the extra stuff. Works with `it("description")` too!

Reply:
!!! Thats way better!!!

Reply:
Still needed a no-op function, but I removed the useless `it`.